x/reports/types/permissions.go

```diff
@@ -0,0 +1,21 @@
+package types
+
+// DONTCOVER
+
+import (
+	subspacestypes "github.com/desmos-labs/desmos/v3/x/subspaces/types"
+)
+
+var (
+	// PermissionReportContent allows users to report contents
+	PermissionReportContent = subspacestypes.RegisterPermission("report_content")
```

Comment:
The underline should be replaced with space to have same convention as other modules.